src/lib/viewers/controls/settings/SettingsDropdown.scss

```diff
@@ -0,0 +1,86 @@
+@import './styles';
+
+$settings-listbox-spacing: 5px;
```

Comment:
Since this is a global variable, can we prefix it with `bp` and use SUIT naming conventions?